### ISSUE TITLE
how to set a proxy to access repository resource

### ISSUE THREAD

**Issue by nemoyin** [openresty/opm#43]
rt

**Comment by detailyang** [comment]
Hello.
If you want to use proxy server to download package in cli, you can set the `http_proxy` or `https_proxy` env before using opm as the following:

```bash
 export http_proxy=http://10.203.0.1:5187/
 export https_proxy=$http_proxy
 export ftp_proxy=$http_proxy
 export rsync_proxy=$http_proxy
 export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com"
```

These is a document [proxy_settings](https://wiki.archlinux.org/index.php/proxy_settings) which is useful to you:)

**Comment by nemoyin** [comment]
thanks, i will try

**Comment by agentzh** [comment]
This is officially documented in opm:

https://github.com/openresty/opm#http-proxy-support

**Comment by kalahari** [comment]
I was unable to get this to work in my CI system until I applied the fix for #56
